doc/modules/ensemble.rst

```diff
@@ -165,20 +165,20 @@ in bias::
     >>> X, y = make_blobs(n_samples=10000, n_features=10, centers=100,
     ...     random_state=0)
 
-    >>> clf = DecisionTreeClassifier(max_depth=None, min_samples_split=1,
```

Comment:
noob question: But should `min_samples_split < 2` raise an error?

What does split mean when there is only one sample in the node?